e2e/ansible/playbooks/feature/snapshots/simple-volume/snapshot.yml

```diff
@@ -1,54 +1,41 @@
 ---
 - hosts: localhost
 
-  vars_files: 
+  vars_files:
     - snapshot-vars.yml
 
   tasks:
    - block:
+       - include: simple-volume-prerequisites.yml
 
-       - name: Get maya-apiserver pod name
-         shell: source ~/.profile; kubectl get pods --selector=name=maya-apiserver
-         args:
-           executable: /bin/bash
-         register: result_name
-         delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+       - include_tasks: "{{ansible_env.HOME}}/{{utils_path}}/deploy_check.yml"
+         vars:
+            ns: openebs
+            app: maya-api
```

Comment:
Since we are updating the helm to use stable charts, maya-api server is deprecated and now it will be openeb-api server. Can you make the changes accordingly?